Demo > Show failure always on top

ruudk opened the pull request:
When working on small screens, it's very easy to mis the error messages as they are located under the Process button.

This is also why it took me some time to know that the Demo was broken (#270).

This will improve it.

<img width="1062" alt="Screenshot 2021-02-15 at 11 56 17@2x" src="https://user-images.githubusercontent.com/104180/107937885-da135b80-6f84-11eb-8f0d-4b809cc96ea1.png">

TomasVotruba commented:
Looks good!


Could you remove the extra space under the text? 
![Demo-Show-failure-always-on-top-by-ruudk-·-Pull-Request-273-·-rectorphp-getrector-org](https://user-images.githubusercontent.com/924196/107958538-713adc00-6fa2-11eb-9569-8558e9dcec1e.png)

So it looks esteticaly

ruudk commented:
I did not change the styling of the error message. For the screenshot I removed the original error and replaced it with this text because I did not wanted to share the real error message I was getting (pssssst: #249)

ruudk commented:
Ok, now that I got the demo working again, this is how it look:
<img width="1027" alt="Screenshot 2021-02-15 at 16 11 12@2x" src="https://user-images.githubusercontent.com/104180/107963534-9599b700-6fa8-11eb-937d-7b82f1e8a8d6.png">

Added a trim, so it looks like:
<img width="1036" alt="Screenshot 2021-02-15 at 16 11 49@2x" src="https://user-images.githubusercontent.com/104180/107963545-99c5d480-6fa8-11eb-974c-eea16c7d8946.png">


Better :) ?

TomasVotruba commented:
Purfect, thank you :hugs:
